src/globemap.ts

```diff
@@ -320,9 +318,37 @@ module powerbi.extensibility.visual {
                 selected: null
             };
         }
-
+        private addAnInstanceToEnumeration(
+            instanceEnumeration: VisualObjectInstanceEnumeration,
+            instance: VisualObjectInstance): void {
+
+            if ((instanceEnumeration as VisualObjectInstanceEnumerationObject).instances) {
+                (instanceEnumeration as VisualObjectInstanceEnumerationObject)
+                    .instances
+                    .push(instance);
+            } else {
+                (instanceEnumeration as VisualObjectInstance[]).push(instance);
+            }
+        }
         public enumerateObjectInstances(options: EnumerateVisualObjectInstancesOptions): VisualObjectInstance[] | VisualObjectInstanceEnumerationObject {
-            return GlobeMapSettings.enumerateObjectInstances(this.settings || GlobeMapSettings.getDefault(), options);
+            let instances: VisualObjectInstanceEnumeration = GlobeMapSettings.enumerateObjectInstances(this.settings || GlobeMapSettings.getDefault(), options);
+            switch (options.objectName) {
+                case 'dataPoint': if (this.data && this.data.seriesDataPoints) {
+                    for (let i: number = 0; i < this.data.seriesDataPoints.length; i++) {
+                        let dataPoint: GlobeMapSeriesDataPoint = this.data.seriesDataPoints[i];
+                        this.addAnInstanceToEnumeration(instances, {
+                            objectName: 'dataPoint',
+                            displayName: dataPoint.label,
+                            selector: ColorHelper.normalizeSelector((dataPoint.identity as ISelectionId).getSelector()),
+                            properties: {
+                                fill: { solid: { color: dataPoint.color } }
+                            }
+                        });
+                    }
+                }
+                    break;
```

Comment:
Please remove unnecessary indent before the ```break```. It should be on the same level like the curly bracket on previous line.